### ISSUE TITLE
Exercise 4 - step 14

### ISSUE THREAD

**Issue by knafil** [NorhanKhaled/SkillsAcademy-IOT#75]
When I try to deploy, I got the problem that : "URL workspaceid parameter '917d91afc5be' is not a valid GUID."

**Comment by knafil** [comment]
Where can we got the Workspace ID ?

**Comment by fadesa** [comment]
I have exactly the same problem @knafil . did you solve the problem ?
![image](https://user-images.githubusercontent.com/52282074/88653377-e062e880-d0c3-11ea-90cd-15ff6cd6154a.png)

**Comment by knafil** [comment]
Yes I solve it. The workspace Id to use is the same as the Skill Id generated in part 2 of Exercise 4.

**Comment by fadesa** [comment]
thanks. it is working !
